src/java.base/share/classes/java/security/KeyStoreSpi.java

```diff
@@ -316,6 +316,7 @@ public abstract void engineStore(OutputStream stream, char[] password)
      *          algorithm could not be found
      * @throws    CertificateException if any of the certificates included in
      *          the keystore data could not be stored
+     * @throws    UnsupportedOperationException if this method is not supported
```

Comment:
We might be able to get it more clear about the behavior.  For example, adding an implSpec tag, like:
```
     * @implSpec The default implementation throws
     *           an {@link UnsupportedOperationException}.
     ...
     * @throws UnsupportedOperationException if the implementation
     *         has not overridden this method

```

Reply:
tag implSpec added. Thanks for the review.